tests/google/cloud/triggers/test_bigquery.py

```diff
@@ -92,7 +92,7 @@ async def test_bigquery_insert_job_op_trigger_success(mock_job_status):
     # so we validate for length of task to be 1
     assert len(task) == 1
 
-    assert TriggerEvent({"status": "success", "message": "Job completed"}) in task
+    assert TriggerEvent({"status": "success", "message": "Job completed", "jobId": TEST_JOB_ID}) in task
```

Comment:
```suggestion
    assert TriggerEvent({"status": "success", "message": "Job completed", "job_id": TEST_JOB_ID}) in task
```

Reply:
same question and dependent on above comment https://github.com/astronomer/astronomer-providers/pull/290#discussion_r861715507

Reply:
changed it to snake case across all needed references.